docs/example-scenario/apps/ecommerce-scenario.md

```diff
@@ -0,0 +1,152 @@
+---
+title: E-Commerce on Azure Platform as a Service
+description: Proven scenario of hosting an e-commerce site on Azure
+author: masonch
+ms.date: 7/13/18
+---
+# E-Commerce on Azure Platform as a Service
+
+Azure Platform-as-a-Service (PaaS) enables you to deploy enterprise grade e-commerce applications, and lets you adapt to the size and seasonality of your business. When demand for your products or services takes off — predictably or unpredictably — you can be prepared to handle more customers and more transactions automatically. Additionally, take advantage of cloud economics by paying only for the capacity you use. In short, focus on your sales and leave the infrastructure management to your cloud provider.
+
+This document will help you will learn about various Azure PaaS components and considerations used to bring together to deploy a sample e-commerce application, *Relecloud Concerts*, an online concert ticketing platform.
```

Comment:
This is the only mention of the company name, do we need this? 

Alternatively, search brought me to https://www.relecloudconcerts.com, I assume that's a deployment of the sample? Maybe link to this?

Reply:
I think it's better to remove it, unless we know for sure it's a public reference